tests/hub-stack-deploy.bats

```diff
@@ -0,0 +1,156 @@
+#!./bats/bin/bats
+# Copyright (c) 2023 EPAM Systems, Inc.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+gen_stack() {
+    mkdir -p "$1/component"
+    cat > "$1/.env" << EOF
+HUB_DOMAIN_NAME="iamtest"
+HUB_STACK_NAME="iamtest"
+HUB_INTERACTIVE="0"
+HUB_FILES="$1/hub.yaml"
+HUB_STATE="$1/hub.state"
+HUB_ELABORATE="$1/hub.elaborate"
```

Comment:
Need to add `HUB_DEPLOY_PROFILE=local` here, otherwise the test fails with `Cannot find deployment profile:`
Lets implicitly set the profile to local if not provided by the user?

Reply:
Thanks! added with last commit